packages/dp-auctions-lib/src/index.js

```diff
@@ -9,7 +9,7 @@ const pTap = require('p-tap').default
 const { findToken } = require('./token-list')
 const dpaAbi = require('./abi.json')
 
-const DPA_ADDRESS = '0x164D41ceB60489D2e054394Fc05ED1894Db3898a' // Chain ID 1
+const DPA_ADDRESS = '0xD8039420f1A61D0930127B89ab7Be23a7f21cd77' // Chain ID 743111
```

Comment:
can we move this into an env variable? As for Hemi mainnet, this contract will change too, and at some point we may have both PF (testnet and mainnet) deployed in 2 different URLs. No need for multichain in one instance, but this will be defined in build time

Reply:
Made this configurable via envs here: https://github.com/hemilabs/pure.finance/pull/2/commits/1c0a792be1d0b294cee93ac808edb62ac1b226a9